upf/upf_pfcp_api.c

```diff
@@ -47,6 +47,8 @@
 
 #include <vlib/unix/plugin.h>
 
+#include <vnet/fib/fib_path_list.h>
+
```

Comment:
this should go a few lines up, right after the include <vnet/ip/ip6_hop_by_hop.h>